test/image/image_info.json

```diff
@@ -1,1612 +1 @@
10\u00ba\u00b2\u000f\u000f\u0000\u00c7\u0000\u0000\u00c5\u0000\u00fc\u00fc\u00fc\u0000\u00c3\u0000\u0004\u0004\u0097\u0000\u00c1\u0000-w-\u00fa\u00fa\u00fa\u0000\u00bf\u0000\u0000\u0000h\u0000\u00bd\u0000\u00da\u00d4\u00d4\u0010\u0010{\t\u00ad\t\u00f8\u00f8\u00f8\u0000\u00b9\u0000\u0000\u00b7\u0000\u009c\u009c\u00a1\u00f6\u00f6\u00f6\f\u00a1\f\u0000\u00b3\u0000\u0000\u00b1\u0000\u0010\u0091\u0010\u0000\u00ab\u0000\u00f2\u00f2\u00f2\u0000\u00a9\u0000\u0000\u00a7\u0000\u00ee\u00ee\u00ee\u00ec\u00ec\u00ec\u0000\u0097\u0000\u00ea\u00ea\u00ea\u0000\u0000k\u0000\u008d\u0000\u0000\u008b\u0000\u0000\u0089\u0000\u0000\u0087\u0000111\u0019\u0019*\u0007\u0007@\u0012\u0012\u00ae\t\t\u00ee\u00d3\u00d6\u00d3\u00cc\u00cc\u00cc\u0093\u0000\u0000\u00a3**\u00c6\u00c6\u00c6\u00c4\u00c4\u00c4\u00cc\u0000\u0000\u00d3\n\n\u0081kkx>>\u0003\u0003m\u0000\u0000\u009f\n\nVFFHAAMs]]Q\u0007\u0007\u00a8\u00a8\u00a8\u00ed\u00ed\u00ef\u00a0\u00a0\u00a0\u009e\u009e\u009e\u009c\u009c\u009c\u0000\u0000\u00b9\u009a\u009a\u009a\u00b2\u0000\u0000\u0098\u0098\u0098\u00eb\u0000\u0000\u0096\u0096\u0096\u009e\u0000\u0000\u0094\u0094\u0094\u0013\u0013\u001c\u0092\u0092\u0092\u0090\u0090\u0090\u008e\u008e\u008e\u0000\u0000\u0091\u008a\u008a\u008a\u0088\u0088\u0088v\u00a2v\u0085\u0088\u0085\u0086\u0086\u0086\u0084\u0084\u0084x\u000f\u000f\u0082\u0082\u0082~\u0082~~~~|||zzzJ\u00c0Jvvvtttrrrppp\u0098\u0000\u0000nnnjpjzffjjj\u0084\u0000\u0000D\u00a2DK\u0096Kadabbb\u0000\u00f4\u00008\u00988\\\\\\ZZZ\u0000\u00e6\u0000\u0002\u0002]BxBVVV\u0000\u0000\u00f0.\u0094.GhG11`\u0000\u00d8\u0000RRRPPP\u0000\u00ce\u0000!\u0098!\u0014\u0014m\u0000\u00c8\u0000LLL\u00fd\u00fd\u00fd\u0000\u00c6\u0000\u0000\u0000\u009dokk\u00ea\u000e\u000eHHH\u0000\u00bc\u00004f4\u0002\u0002\u00f5SS`\u0000\u00b0\u0000\u0000\u00ae\u0000\u00b7\u0000\u0000\u0000\u00a4\u0000\u00ef\u00ef\u00ef\u0000\u00a2\u0000\u008fOO\u009544\u0000\u0098\u0000\u0000\u0096\u0000\u0000\u0092\u0000\u00e7\u00e7\u00e7\u0000\u008c\u0000\u00e3\u00e3\u00e3\u0000\u0080\u0000\u0092\u0006\u0006\u0001\u0001odVY\u0000\u0000dNNk\u00cf\u00cf\u00cf\u00dd\u00dd\u00e0\u0000\u0000{\u00c9\u00c9\u00c9\u0016\u0016A!!YWWYDDx\u00bf\u00bf\u00bf\u00eb\u00dd\u00dd--T\n\n`\u0000\u0000\u00ff\u00bf}}", "w": 32, "h": 32, "cs": "Indexed", "bpc": 8, "f": "FlateDecode", "dp": "/Predictor 15 /Colors 1 /BitsPerComponent 8 /Columns 32", "trns": ""}, "z00n2c08.png": {"w": 32, "h": 32, "cs": "DeviceRGB", "bpc": 8, "f": "FlateDecode", "dp": "/Predictor 15 /Colors 3 /BitsPerComponent 8 /Columns 32", "trns": ""}, "z03n2c08.png": {"w": 32, "h": 32, "cs": "DeviceRGB", "bpc": 8, "f": "FlateDecode", "dp": "/Predictor 15 /Colors 3 /BitsPerComponent 8 /Columns 32", "trns": ""}, "z06n2c08.png": {"w": 32, "h": 32, "cs": "DeviceRGB", "bpc": 8, "f": "FlateDecode", "dp": "/Predictor 15 /Colors 3 /BitsPerComponent 8 /Columns 32", "trns": ""}, "z09n2c08.png": {"w": 32, "h": 32, "cs": "DeviceRGB", "bpc": 8, "f": "FlateDecode", "dp": "/Predictor 15 /Colors 3 /BitsPerComponent 8 /Columns 32", "trns": ""}}
```

Comment:
Could you please preserve the original JSON formatting?
Currently the diff does not make it possible to understand the changes you made...

Reply:
Yes you are right, I didn't thought about the formatting